gradle/libs.versions.toml

```diff
@@ -23,6 +23,7 @@ springBoot = "3.3.3"
 springDoc = "2.6.0"
 wsdl4j = "1.6.3"
 wsdl2java = "2.0.2"
+xroadLibs = "7.6.0-SNAPSHOT"
```

Comment:
Yes Snapshots dependencies shouldn't be used in release versions.
But since this is the first version we have of the configuration-client and this is develop branch of the catalog service, I guess it's temporarily OK.

Good point with the release process. Will check and come back to you with this.